### ISSUE TITLE
problems installing awscli on OSX

### ISSUE THREAD

**Issue by Rovel** [asdf-vm/asdf#519]
### Steps to reproduce

1. install latest python 3.7
2. `pip3 install awscli --upgrade --user`
3. try to find the correct bin path to export on profile

### Expected behavior

You should be able to run awscli commands, like `aws configure`

### Actual behavior

aws not found

### Environment

**OS**: OSX 14.4 Mojave

**asdf version**: latest

**Comment by Stratus3D** [comment]
This issue is specific to asdf-python. Please open an issue there - https://github.com/danhper/asdf-python

**Comment by Rovel** [comment]
Hi, 

Found the correct path:

`export PATH=/Users/$USER/.local/bin:$PATH`

Hope this helps someone in the future

**Comment by danhper** [comment]
I suspect that you simply forgot to run `asdf reshim python`
asdf should not install anything in `~/.local/bin` so the executable there might have been installed through another method.

**Comment by Rovel** [comment]
I tried all sort of things including reshim, but the only one that worked was that, the problem is that was looks for the python there and asdf installs elsewhere, so setting the path solves

**Comment by joneslee85** [comment]
in theory asdf should not install anything to `~/.local/bin` but the reality is that `awscli` does get installed there.

So the suggestion by @Rovel is considered a correct one

**Comment by tayler-jones** [comment]
I had a similar issue, but found that `pip install awscli` (no --user flag) installs it to the correct location given by $(asdf where python). `aws` worked fine after a reshim. 

When I provide --user flag, it installs to ~/.local/bin/ and I get the same issue described in the original post. Since asdf is installed to $HOME/.asdf and $USER owns the installed binaries, it seems fine to leave off the --user flag.

OSX Mojave 10.14.6 w/ current master of asdf-vm and asdf-python

**Comment by tayler-jones** [comment]
I came across this discussion after running into the same issue, but I found a different solution. Adding this comment in case anyone else finds it, too.

`pip install awscli` without the --user flag installs the aws binaries to the correct location under $(asdf where python). The `aws` commands work as expected after running `asdf reshim python` and reloading my shell (zsh).

Note that pip installs the libraries to `~/.local/lib/pythonX.Y/site-packages` whether or not --user is provided.

Mojave 10.14.6 with current asdf-vm and asdf-python
